controllers/application.go

```diff
@@ -71,7 +68,7 @@ func (c *ApiController) GetApplications() {
 			return
 		}
 
-		applications := object.GetMaskedApplications(app, userId)
+		applications, _ := object.GetMaskedApplications(app)
```

Comment:
handle error

Reply:
The GetMask***() after  #1836, use ...error as the last parameter. If you don't pass the err when call the function, it will not return error that not equal nil. I'm not sure if it is a good design. But in fact, now similar uses are redundant and repetitive handling err.